src/NuGet/BuildNuGets.csx

```diff
@@ -33,6 +34,19 @@ var BuildingReleaseNugets = IsReleaseVersion(BuildVersion);
 var NuspecDirPath = Path.Combine(SolutionRoot, "src/NuGet");
 var OutDir = Path.GetFullPath(Args[2]).TrimEnd('\\');
 
+// In developer builds the commit sha is <developer build>. Need to make sure
+// we escape the values so that 
+var CommitSha = Args[3].Replace("<", "").Replace(">", "");
+var CommitIsDeveloperBuild = CommitSha == "<developer build>";
+if (!CommitIsDeveloperBuild && !Regex.IsMatch(CommitSha, "[A-Fa-f0-9]+"))
+{
+    Console.WriteLine("Invalid Git sha value: expected <developer build> or a valid sha");
+    Environment.Exit(1);
+}
+var CommitPathMessage = CommitIsDeveloperBuild
+    ? $"This package was built from the source at https://github.com/dotnet/roslyn/commit/{CommitSha}"
+    : "This an unofficial build from a developer's machine";
```

Comment:
Shouldn't the ternary be reversed? i.e. `CommitIsDeveloperBuild ? "This is an unofficial build[...]" : $"This package was build from[...]"`